prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/FhirPathUtils.kt

```diff
@@ -104,10 +105,20 @@ object FhirPathUtils : Logging {
             logger.error("${e.javaClass.name}: FHIR path could not find a specified field in $expression.")
             emptyList()
         }
+
+        if (expression.equals("%resource.postalCode.getStateFromZipCode()")) {
+            if (retVal.first().toString().isEmpty()) {
```

Comment:
This is the highest piece of the stack where we are aware that the zip code lookup failed. Throwing an exception here _should_ only interrupt the processing for a single element (ie the state associated with the zip code... which has already failed). I haven't thoroughly proven that this won't potentially introduce new issues yet.